build.cmd

```diff
@@ -403,6 +403,7 @@ set "__BuildLog=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.l
 set "__BuildWrn=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
 set "__BuildErr=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
 set __msbuildLogArgs=^
+/p:UseOpenSourceSign=false ^
```

Comment:
I dont believe you want to disable OSS signing for good. This argument should be added based upon a conditional argument passed to the script which would, for now, only come from the VSO build invocation.